src/declaration.d.ts

```diff
@@ -0,0 +1 @@
+declare module '*.png';
```

Comment:
Maybe change the file name to `global.d.ts` instead. Also, add a blank line at the end of the file

Reply:
ok